src/tools/wasm2js.cpp

```diff
@@ -812,17 +833,20 @@ void AssertionEmitter::emit() {
 
   Builder wasmBuilder(sexpBuilder.getModule());
   Name asmModule = std::string("ret") + ASM_FUNC.str;
+  // Track the last built module.
+  Module wasm;
   for (size_t i = 0; i < root.size(); ++i) {
     Element& e = *root[i];
     if (e.isList() && e.size() >= 1 && e[0]->isStr() &&
         e[0]->str() == Name("module")) {
+      wasm.~Module();
+      new (&wasm) Module();
```

Comment:
```suggestion
      ModuleUtils::clearModule(wasm);
```
(that literally does those two lines)

Reply:
Ah, I had thought we had a function for doing that, but I was looking in the wrong place. Thanks!